src/features/rewards/tableDonation/index.tsx

```diff
@@ -89,39 +92,41 @@ export default class TableDonation extends React.PureComponent<Props, {}> {
       return
     }
 
-    return rows.map((row: DetailRow): Row => {
-      return {
-        content: [
-          {
-            content: (
-              <StyledLink href={row.url} target={'_blank'}>
-                <Profile
-                  title={row.profile.name}
-                  provider={row.profile.provider}
-                  verified={row.profile.verified}
-                  src={row.profile.src}
+    return rows.map(
```

Comment:
please don't change format of the code as it's harder to track what was actually changed